### DISCUSSION TITLE
Window snapping in Windows with AquaSnap?

### DISCUSSION THREAD

**Discussion by zmoon** [keepassxreboot/keepassxc#9693] [Q&A] [answered]
In Windows, the app doesn't seem to respond to window snapping/moving/resizing commands, e.g. <kbd>win</kbd>+<kbd>left</kbd>. Is this something that is possible to configure or fix? Or is it perhaps a limitation of the GUI framework?

**Comment by droidmonkey** [comment, marked as answer]
Works perfectly fine for me.

**Reply by zmoon** [reply]
Ah ok sorry I see the example I gave does work for me also if I disable AquaSnap. But with AquaSnap on it doesn't. Based on [what they say](https://www.nurgo-software.com/company/14-support) about Adobe Reader, maybe it is a sandboxing thing.

**Reply by droidmonkey** [reply]
Sure if you put some third party software in the mix then YMMV
